### ISSUE TITLE
Windows VS Code Visual Editor: switching between apps loses coursor

### ISSUE THREAD

**Issue by mobius-eng** [quarto-dev/quarto#131]
In visual editor when switching from VS Code to another app and back the coursor is lost (it almost looks like Vim's edit mode since arrow keys just scroll the text). This happens if the swith is performed via Alt-Tab or via mouse click. On Alt-Tab switch, the menu gets the focus (as if Alt alone was pressed). The issue is specific to visual mode, in source mode everything works as expected.

**Comment by jjallaire** [comment]
Yes, we've seen this but there doesn't seem to be a way to work around it (I think its a vagary of how VS Code handles custom editors as opposed to text editors). We also don't get notified when the windows is deactivated/reactivated so there isn't a basis for us to implement a workaround :-(
